browser/js/history/history.state.js

```diff
@@ -0,0 +1,11 @@
+'use strict'
+
+app.config(function ($stateProvider) {
+
+	// Register our *history* state.
+	$stateProvider.state('history', {
+		url: '/history',
+		controller: 'History',
```

Comment:
naming convention is title-cased name of state + 'ctrl'. in this case it'd be `HistoryCtrl`